engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -136,7 +136,7 @@ class WorkflowActor(workflowId: WorkflowId,
                     serviceRegistryActor: ActorRef)
   extends LoggingFSM[WorkflowActorState, WorkflowActorData] with ActorLogging with KnowsWhatTimeItIs {
 
-  val tag = self.path.name
+  val tag = s"WorkflowActor [UUID(${workflowId.shortString})]"
```

Comment:
ToL: It would be really nice if the query APIs also accepted these short strings. I might add a ticket for that